ConcernsCaseWork/ConcernsCaseWork/Security/IRbacManager.cs

```diff
@@ -6,9 +6,9 @@ namespace ConcernsCaseWork.Security
 {
 	public interface IRbacManager
 	{
-		IList<string> GetDefaultUsers();
-		Task<IDictionary<string, RoleClaimWrapper>> GetUsersRoles();
+		Task<IList<string>> GetSystemUsers(params string[] excludes);
+		//Task<IDictionary<string, RoleClaimWrapper>> GetUsersRoles();
```

Comment:
Similar to the question about 'TODO' - are we happy to push commented out code into prod?

Reply:
removed